iceoryx_hoofs/concurrent/buffer/include/iox/detail/mpmc_lockfree_queue/mpmc_index_queue.inl

```diff
@@ -36,7 +36,7 @@ inline IndexQueue<Capacity, ValueType>::IndexQueue(ConstructEmpty_t) noexcept
 }
 
 template <uint64_t Capacity, typename ValueType>
-IndexQueue<Capacity, ValueType>::IndexQueue(ConstructFull_t) noexcept
+MpmcIndexQueue<Capacity, ValueType>::MpmcIndexQueue(ConstructFull_t) noexcept
```

Comment:
For most method definitions, the required `inline` keyword is missing. Could you please add it.